client/components/AddTestToQueueWithConfirmation/index.jsx

```diff
@@ -47,12 +51,12 @@ function AddTestToQueueWithConfirmation({
 
     const alreadyHasBotInTestPlanReport = useMemo(
```

Comment:
We should update the name of this variable, as well, since we're no longer specifically considering bot-assigned Test Plan Runs.

Reply:
You're right--I don't know how I missed that!